Install errors in Atom 1.2.4 / gyp

Hello,

Tried to reinstall Atom 1.2.4 in Windows with autocomplete-polymer, but now I am getting install errors related to node-gyp.

Did a lot of research, but can't seem to find out how to solve this or even explain this.
I think it could be related to the scanal/git-utils dependencies as can be seen in the stack trace.
Any ideas or can someone reproduce this? Thanks.

Further environment settings:

apm  1.4.0
npm  2.13.3
node 0.10.40
python 2.7.10
git 2.6.0.windows.1
visual studio 2013

------------------------------------

Stack trace:

> git-utils@3.0.1 install E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils
> node-gyp rebuild

E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils>if not defined npm_config_node_gyp (node "C:\Users\danielm\AppData\Local\atom\app-1.2.4\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild ) 
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppBuild.targets(396,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\libgit2.vcxproj]
  attr.c
  attr_file.c
  attrcache.c
  blame_git.c
  blame.c
  blob.c
  branch.c
  buf_text.c
  buffer.c
  cache.c
  checkout.c
  cherrypick.c
  clone.c
  commit.c
  commit_list.c
  config.c
  config_cache.c
  config_file.c
  crlf.c
  date.c
  delta-apply.c
  delta.c
  diff.c
  diff_driver.c
  diff_file.c
  diff_patch.c
  diff_print.c
  diff_stats.c
  diff_tform.c
  diff_xdiff.c
  errors.c
  fetch.c
  fetchhead.c
  filebuf.c
  fileops.c
  filter.c
  fnmatch.c
  global.c
  graph.c
  hash.c
  hashsig.c
  ident.c
  ignore.c
  index.c
  indexer.c
  iterator.c
  merge.c
  merge_file.c
  message.c
  mwindow.c
  netops.c
  notes.c
  object.c
  object_api.c
  odb.c
  odb_loose.c
  odb_mempack.c
  odb_pack.c
  oid.c
  pack-objects.c
  pack.c
  path.c
  pathspec.c
  pool.c
  posix.c
  pqueue.c
  push.c
  refdb.c
  refdb_fs.c
  reflog.c
  refs.c
  refspec.c
  remote.c
  repository.c
  reset.c
  revert.c
  revparse.c
  revwalk.c
  settings.c
  sha1_lookup.c
  signature.c
  sortedcache.c
  stash.c
  status.c
  strmap.c
  submodule.c
  sysdir.c
  tag.c
  thread-utils.c
  trace.c
  transport.c
  tree-cache.c
  tree.c
  tsort.c
  util.c
  vector.c
  zstream.c
  cred.c
  cred_helpers.c
  git.c
  http.c
  local.c
  smart.c
  smart_pkt.c
  smart_protocol.c
  ssh.c
  winhttp.c
  xdiffi.c
  xemit.c
  xhistogram.c
  xmerge.c
  xpatience.c
  xprepare.c
  xutils.c
  hash_generic.c
  dir.c
  error.c
  findfile.c
  map.c
  posix_w32.c
  precompiled.c
  pthread.c
  utf-conv.c
  w32_util.c
  regex.c
  libgit2.vcxproj -> E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\Release\\libgit2.lib
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppBuild.targets(396,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\zlib.vcxproj]
  adler32.c
  crc32.c
  deflate.c
  inffast.c
  inflate.c
  inftrees.c
  trees.c
  zutil.c
  zlib.vcxproj -> E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\Release\\zlib.lib
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppBuild.targets(396,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\http_parser.vcxproj]
  http_parser.c
  http_parser.vcxproj -> E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\Release\\http_parser.lib
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppBuild.targets(396,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
  repository.cc
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(262): error C2995: 'v8::Local<T> _NanEnsureLocal(v8::Local<T>)' : function template has already been defined [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
          E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(255) : see declaration of '_NanEnsureLocal'
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(660): error C3083: 'smalloc': the symbol to the left of a '::' must be a type [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(660): error C2039: 'FreeCallback' : is not a member of 'node' [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(660): error C2061: syntax error : identifier 'FreeCallback' [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(664): error C2065: 'callback' : undeclared identifier [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(664): error C2065: 'hint' : undeclared identifier [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(671): error C2665: 'node::Buffer::New' : none of the 4 overloads could convert all the argument types [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
          C:\Users\danielm\.atom\.node-gyp\.node-gyp\0.34.0\src\node_buffer.h(53): could be 'v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate *,char *,size_t)'
          C:\Users\danielm\.atom\.node-gyp\.node-gyp\0.34.0\src\node_buffer.h(41): or       'v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate *,v8::Local<v8::String>,node::encoding)'
          while trying to match the argument list '(v8::Isolate *, const char *, uint32_t)'
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(675): error C2664: 'v8::Local<v8::Object>::Local(const v8::Local<v8::Object> &)' : cannot convert argument 1 from 'v8::MaybeLocal<v8::Object>' to 'const v8::Local<v8::Object> &' [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
          Reason: cannot convert from 'v8::MaybeLocal<v8::Object>' to 'const v8::Local<v8::Object>'
          No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(682): error C2039: 'Use' : is not a member of 'node::Buffer' [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]
E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\node_modules\nan\nan.h(682): error C3861: 'Use': identifier not found [E:\temp\apm-install-dir-115113-4568-y6tsga\node_modules\autocomplete-polymer\node_modules\scandal\node_modules\git-utils\build\git.vcxproj]

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\danielm\\AppData\\Local\\atom\\app-1.2.4\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\danielm\\AppData\\Local\\atom\\app-1.2.4\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\danielm\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\danielm\\.atom\\.apmrc" "install" "E:\\temp\\d-115113-4568-oz2ty2\\package.tgz" "--target=0.34.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE

npm ERR! git-utils@3.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the git-utils@3.0.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the git-utils package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls git-utils
npm ERR! There is likely additional logging output above.

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "esprima": "^2.2.0",
-    "scandal": "2.0.0",
+    "scandal": "2.2.0",
     "temp": "0.8.1",
     "estraverse": "^4.1.0"
   },

